src/general/OptionsMenu.cs

```diff
@@ -1002,6 +1002,11 @@ private void OnTutorialsOnNewGameToggled(bool pressed)
     private void OnCheatsToggled(bool pressed)
     {
         Settings.Instance.CheatsEnabled.Value = pressed;
+        if (!pressed)
+        {
+            CheatManager.DisableAllCheats();
```

Comment:
One thing this reminds me of: I think these 2 methods should have a single method like `TurnOffCheats` that calls them both, because I think that that should get called when a new game is started or a save is loaded. Because I don't think cheats should stay on if you start a new game or load a different save.